docs/Architecture.md

```diff
@@ -67,24 +67,30 @@ soon as it can, and all of the real work will be done as that user.
 
 ### Message
 
-### Policy
-
 ### Profile
 
->
-> TODO: Come up with a better name to distinguish it from *config profiles*.
->
+A profile is a configuration for Zonemaster Engine; see the [profiles
+overview] for context.
+Zonemaster Backend allows administrators to [configure the set of
+available profiles].
 
-### Config profile
+Each available profile has a [profile name].
+Each one is based on the [Zonemaster Engine default profile],
+Each one (with the possible exception of `default`) has a [profile file]
+with overrides to the Zonemaster Engine default profile.
 
```

Comment:
The default Engine default profile is always loaded. Any profile selected here will only override the parameters defined in the overriding profile file.